etools-loading-behavior.html

```diff
@@ -1,9 +1,18 @@
 <link rel="import" href="etools-loading.html">
+<link rel="import" href="scripts/lodash.html">
```

Comment:
this points to a lodash script that's not imported from this repo.. and i don';t see a dependency req for it..
`<script src="../../lodash/lodash.min.js"></script>`
assumes that you have that in the project... since this is an independent component it needs to have all the code that it requires to run inside the element... in this case I'm not sure it's a good idea...
can you work around the lodash requirement? or should we put it as a req in this repo?